packages/itwin/property-grid/src/components/FilteringPropertyGrid.tsx

```diff
@@ -112,26 +118,53 @@ implements IPropertyDataProvider, IDisposable {
   }
 }
 
-interface FilteringPropertyGrid
+/**
+ * Properties for rendering a `FilteringPropertyGrid`
+ */
+interface FilteringPropertyGridProps
   extends VirtualizedPropertyGridWithDataProviderProps {
   filterer: PropertyDataFiltererBase;
   autoExpandChildCategories?: boolean;
 }
 
+/**
+ * Calls the `usePropertyDataProviderWithUnifiedSelection` hook before rendering a `FilteringPropertyGrid`
+ */
 export const FilteringPropertyGridWithUnifiedSelection = (
-  props: FilteringPropertyGrid
+  props: FilteringPropertyGridProps
 ) => {
   const localizations = useMemo(() => ({
     tooManySelected: PropertyGridManager.translate(
       "context-menu.selection.too-many-elements-selected"
     ),
-    noneSelected: PropertyGridManager.translate(
```

Comment:
why was this removed? isnt this needed for ui-1?

Reply:
I couldn't find any reference to it in the code. If it's referenced in some other way where it wouldn't show up in a search I can add it back.

Reply:
Verified with the other design review devs about removing this string. They said it shouldn't matter for UI 1.0 and since it's not referenced in any of the files it should be removed.